### ISSUE TITLE
Make servo icon match servo position

### ISSUE THREAD

**Issue by ehamner** [CMU-CREATE-Lab/flutter-app-android#74]
The servo icon on the servo relationship label was meant to show the current position of the servo.

![image](https://cloud.githubusercontent.com/assets/11526803/22756542/72c68050-ee15-11e6-9b96-e516be072b32.png)

**Comment by jenncross** [comment]
Similar in implementation to #73 

Duplicate note:
------
This could be implemented one of two ways....

Either:
A. Poll the Flutter constant for the output states (along with sensor values) - which seems complex.
B. Use the current known sensor values and the know links, to compute the output states in the app in parallel to the Flutter hardware.

--- Granted in the long term option A. might be more efficient, once all of the link types are implemented.

**Comment by jenncross** [comment]
Similar to #73 
----

This seems nearly impossible to achieve accurately ... due to:

Maximum polling frequency of the Flutter
State-based value for Change & Cumulative relationships
Rapidly changing values for Amplitude & Frequency relationships
It seems like this is then only feasible on Proportional & Fixed relationships. It will be extremely confusing to have this work on some but not all relationship types.

**_tl;dr: I suspect that this is not likely to be resolved anytime soon._**

**Comment by jenncross** [comment]
Here is an idea, what if we did something like the split LED colors in #159 where we show the two end points of the servo motion range.

I could imagine for example coloring in a wedge of the servo dial image to represent the sweep. Maybe in a light green color?

Thoughts?

**Comment by ehamner** [comment]
Yeah, I think something like that would be nice.

**Comment by jenncross** [comment]
We are going with #196 instead. Closing.
